Sources/Swift/Components/Dates and Timers/IdleTimer/IdleTimer+Gesture.swift

```diff
@@ -25,6 +30,12 @@ extension NotificationCenter.Event {
     public func applicationDidTimeOutUserInteraction(_ callback: @escaping () -> Void) -> NSObjectProtocol {
         observe(UIApplication.didTimeOutUserInteractionNotification, callback)
     }
+
+    /// - See: `IdleTimer.setUserInteractionTimeout(duration:for:)`
+    @discardableResult
+    public func applicationWillTimeOutIdleTimerNotification(_ callback: @escaping () -> Void) -> NSObjectProtocol {
```

Comment:
```suggestion
    public func applicationWillTimeOutUserInteraction(_ callback: @escaping () -> Void) -> NSObjectProtocol {
```
This makes it consistent with the `applicationDidTimeOutUserInteraction` method.